src/main/java/mocacong/server/controller/MemberController.java

```diff
@@ -19,11 +17,21 @@
 public class MemberController {
 
     private final MemberService memberService;
+    private final JwtTokenProvider jwtTokenProvider;
 
     @Operation(summary = "회원가입")
     @PostMapping
     public ResponseEntity<MemberSignUpResponse> signUp(@RequestBody MemberSignUpRequest request) {
         MemberSignUpResponse response = memberService.signUp(request);
         return ResponseEntity.ok(response);
     }
+
+    @Operation(summary = "회원탈퇴")
+    @DeleteMapping
+    public ResponseEntity<String> delete(@RequestHeader("Authorization") String authorizationHeader) {
+        String token = authorizationHeader.replace("Bearer ", "");
+        String userEmail = jwtTokenProvider.getPayload(token);
+        memberService.delete(userEmail);
+        return ResponseEntity.ok("회원 탈퇴를 정상적으로 처리했습니다");
```

Comment:
dto를 담아주지 않고 String으로 담는 것은 json형태가 아니기 때문에 rest api 통신에 적절치 않다고 생각됩니다. 아래와 같은 형식은 어떤가요?
```suggestion
        return ResponseEntity.ok().build();
```

Reply:
저도 저 부분이 거슬렸는데 해결책 알려주셔서 감사합니다! Void로 수정했습니다. 👍